diff --git a/dist/node/index.mjs b/dist/node/index.mjs
index 20707d56ebf387c10608c2f9235321b7befb051c..e5a3fb8cc57dc0067437c58b877320add4e5f2d9 100644
--- a/dist/node/index.mjs
+++ b/dist/node/index.mjs
@@ -870,9 +870,8 @@ const buildBundleFactory = (getOption) => {
 							external: Object.keys(option.globalsPkg2VarName),
 							output: {
 								...rolldownMinify !== void 0 ? { minify: rolldownMinify } : {},
+								...viteConfig.build.rolldownOptions.output,
 								globals: option.globalsPkg2VarName,
-								comments: false,
-								strict: false
 							},
 							experimental: { attachDebugInfo: "none" }
 						},
